source/preparing/text.rst

```diff
@@ -9,6 +9,18 @@ a different text editor may prove to be useful.
 Downloads
 =========
 
+Visual Studio Code
```

Comment:
Why did you move this section to the top?

Reply:
As VS Code has proven to be the more capable editor, with a larger userbase, and a more guaranteed future. Also it seems in the community, it is the more common choice. It's a bit opinionated but I do believe it will provide better development experience.